src/main/java/org/commcare/core/process/CommCareInstanceInitializer.java

```diff
@@ -86,37 +87,24 @@ public InstanceRoot generateRoot(ExternalDataInstance instance) {
             return setupFixtureData(instance);
         } else if (instance.getReference().contains("session")) {
```

Comment:
why are some of these 'startswith' and some 'contains'? Shouldn't they always be an exact match anyway?